src/app/dim-ui/Sheet.tsx

```diff
@@ -251,23 +251,15 @@ export default function Sheet({
           <a
             href="#"
             className={clsx('sheet-close', { 'sheet-no-header': !header })}
-            onClick={handleClose}
+            onClick={triggerClose}
           >
             <AppIcon icon={disabledIcon} />
           </a>
 
-          <div
-            className="sheet-container"
-            onPointerDown={dragHandleDown}
-            onPointerUp={dragHandleUp}
-            onMouseDown={dragHandleDown}
-            onMouseUp={dragHandleUp}
-            onTouchStart={dragHandleDown}
-            onTouchEnd={dragHandleUp}
-          >
+          <div className="sheet-container" onPointerDown={dragHandleDown}>
```

Comment:
Is it OK to lose the `dragHandleUp` binding?

Reply:
I thiiiink so, largely the "dragging has finished" functionality got moved to the `onDragEnd` prop of the motion component. 

Also, the previous callback was the following

```
const dragHandleUp = useCallback(() => (dragging.current = false), []);
```

As we don't need to track locally whether the user is dragging anymore I don't think we need to provide it.